src/SourceBuild/content/build.sh

```diff
@@ -275,10 +275,12 @@ function Build {
         initSourceOnlyBinaryLog="/bl:\"$log_dir/init-source-only.binlog\""
       fi
 
-      "$CLI_ROOT/dotnet" build-server shutdown
+      # --msbuild is added as a temporary workaround - https://github.com/dotnet/source-build/issues/4796
```

Comment:
This is the right change to do here anyway as we only need to stop the msbuild server for the custom sdk resolver to kick in.

Same for the seconds invocation below. I would remove these comments.